nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db/TestJdbcCommon.java

```diff
@@ -102,6 +102,9 @@ public static void setup() throws ClassNotFoundException, SQLException {
         try (final Statement stmt = con.createStatement()) {
             stmt.executeUpdate(createTable);
         }
+
+        // Set default timezone to avoid conflicts with the java.sql.Date library
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
This may resolve the issue, but it's typically not a good idea to change the timezone in this manner.  I have submitted another PR (#3630) which does not alter the default timezone, and shouldn't interfere with other tests if run in parallel.